lib/ferver/controller.rb

```diff
@@ -70,5 +88,12 @@ def current_ferver_path
     def current_full_path
       File.expand_path(current_ferver_path)
     end
+
+    def authorised?(user, password)
```

Comment:
Unused method argument - `password`. If it's necessary, use `_` or `_password` as an argument name to indicate that it won't be used.